README.md

```diff
@@ -1,22 +1,628 @@
+
+a. Install Network File System (NFS) on all instances including the master and each worker. You will need to connect to each VM terminal to do installation. 
+
+`sudo apt update`
+
+`sudo apt install nfs-common -y`
+
+
+<br/>
+
+b. Reconnect to the master instance and navigate into the cloned repository
+
+`cd ~/ofo-argo`
+
+<br/>
+
+c. Set up the persistent volumes (PV) defined by the workflow. You are specifying the read (raw drone imagery on `ofo-share`) and the write (metashape output location)
+
+`kubectl apply -f argo-output-pv.yaml`
+
+`kubectl apply -f ofo-share-pv.yaml`
+
+<br/>
+
+d. Set up persistent volume claims (PVC)
+
+`kubectl apply -f argo-output-pvc.yaml -n argo`
+
+`kubectl apply -f ofo-share-pvc.yaml -n argo`
+
+<br/>
+
+e. Check the PVs
+
+`kubectl get pv`
+
+`kubectl get pvc -n argo`
+
+<br/>
+<br/>
+<br/>
+
+## Run the Workflow
+
+### 1. Declare the ip address of the metashape license server
+
+On the master instance terminal, type:
+
+`export AGISOFT_FLS=<ip_address>:5842`
+
+This variable will only last during the terminal session and will have to be re-declared each time you start a new master terminal. We are not putting the ip address here to prevent unauthorized people from using it. 
+
+<br/>
+
+### 2. Run!!
+
+```
+argo submit -n argo workflow.yaml --watch \
+-p CONFIG_FILE=config2.yml \
+-p AGISOFT_FLS=$AGISOFT_FLS \
+-p RUN_FOLDER=gillan_june27 \
+-p DATASET_LIST=datasets.txt \
+-p DB_PASSWORD=ujJ1tsY9OizN0IpOgl1mY1cQGvgja3SI \
+-p DB_HOST=10.0.205.207 \
+-p DB_NAME=postgres \
+-p DB_USER=postgres
+ 
+```
+
+CONFIG_FILE is the config which specifies the metashape parameters which should be located in `/ofo-share/argo-output`
+
+AGISOFT_FLS is the ip address of the metashape license server
+
+RUN_FOLDER is what you want to name the parent directory of your output
+
+DATSET_LIST is the txt file where you specified the names of the datasets you want to process located at `/ofo-share/argo-output`
+
+The rest of the 'DB' parameters are for logging argo status in a postGIS database. Do not change these. 
+
+<br/>
+
+### 3. Monitor Argo Workflow
+The Argo UI is great for troubleshooting and checking additional logs. You can access it either through the Cacao WebDesktop or ssh from your local terminal.
+
+#### WebDesktop Method
+* In the CACAO interface, launch a WebDesktop for your master instance
+
+<img width="660" alt="Screenshot 2025-06-20 at 9 33 53 AM" src="https://github.com/user-attachments/assets/a3ee09ba-d701-4fa5-97d3-586b4c640dc1" />
+
+<br/>
+<br/>
+
+* Launch a terminal in the WebDesktop 
+
+* Get around a known issue with v3.6.10
+  
+`export GRPC_ENFORCE_ALPN_ENABLED=false`
+
+* Then type this to launch the server UI. 
+
+`argo server --auth-mode server -n argo`
+
+<br/>
+
+* Now go to a browser (firefox) in the WebDesktop and go the address. You may receive a "Connection not secure" error but just bypass it.
```

Comment:
Maybe include how to access it on your local machine too

Reply:
I see this is lower down. This is a minor distinction, but the commands you run don't change based on how you access the terminal (webshell vs. ssh) so maybe these can be combined. And then just leave the distinction between accessing the Argo GUI locally or on the webshell.